curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a7c23e6b511f22ed71197a.md

```diff
@@ -0,0 +1,155 @@
+---
+id: 62a7c23e6b511f22ed71197a
+title: Step 46
+challengeType: 0
+dashedName: step-46
+---
+
+# --description--
+
+You need to wrap the text `Store` in double quotes. Because your string is already wrapped in double quotes, you'll need to escape the quotes around `Store`. You can escape them with a backslash `\`.
```

Comment:
We should add a code example here because beginners won't know intuitively where to place the backslashes.